policyengine_us/variables/gov/states/sc/tax/income/sc_taxable_income.py

```diff
@@ -9,5 +9,5 @@ class sc_taxable_income(Variable):
     unit = USD
     definition_period = YEAR
```

Comment:
could you add a reference? eg to indicate that values can be negative

Reply:
add the reference and changed to a max_ function